readthedocs/oauth/services/githubapp.py

```diff
@@ -349,6 +353,8 @@ def _resync_collaborators(self, gh_repo: GHRepository, remote_repo: RemoteReposi
         Sync collaborators of a repository with the database.
 
         This method will remove collaborators that are no longer in the list.
+
+        See https://docs.github.com/en/rest/collaborators/collaborators?apiVersion=2022-11-28#list-repository-collaborators.
```

Comment:
This is useful :+1: -- I wasn't used to the collaborators wording, but it makes sense.